paymentsheet/src/main/java/com/stripe/android/link/verification/NoOpLinkEmbeddedInteractor.kt

```diff
@@ -0,0 +1,22 @@
+package com.stripe.android.link.verification
+
+import com.stripe.android.lpmfoundations.paymentmethod.PaymentMethodMetadata
+import com.stripe.android.ui.core.elements.OTPSpec
+import com.stripe.android.uicore.elements.OTPElement
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+
+/**
+ * Implementation for surfaces that don't support Link embedded verification.
+ */
+internal class NoOpLinkEmbeddedInteractor : LinkEmbeddedInteractor {
```

Comment:
Using this on the embedded integration since we're not planning on supporting WalletsButton there.